thrift/annotation.go

```diff
@@ -46,14 +46,14 @@ type Annotation interface {
 	//   AnnoKindKeyMapping: KeyMapping interface
 	//   AnnoKindKeyMapping: ValueMapping interface
 	//   AnnoKindOptionMapping: OptionMapping interface
-	Make(ctx context.Context,values []parser.Annotation, desc interface{}) (handler interface{}, err error)
+	Make(ctx context.Context, values []parser.Annotation, desc interface{}) (handler interface{}, err error)
 }
 
 // AnnoID is the unique id of an annotation, which is composed of kind, scope and type:
 //
-//   0xff000000: AnnoKind
-//   0x00ff0000: AnnoScope
-//   0x0000ffff: AnnoType
+//	0xff000000: AnnoKind
```

Comment:
indent format, I perfer using 4-blanks instead of tab

Reply:
ok, I'll put it back in the original format.